src/steps/command/plugins.ts

```diff
@@ -33,4 +35,8 @@ export class PluginsImpl<T> extends Chainable<T> implements Plugins<T> {
         this.plugins.push(plugin);
         return this.parent;
     }
+
+    filter(predicate: (plugin: Plugin) => boolean): Plugin[] {
+        return this.plugins.filter(predicate);
+    }
```

Comment:
We should reuse the original types. This is untested, can you give it a try:
```suggestion
    filter(...args: Parameters<Plugin[]['filter']>) {
      return this.plugins.filter.apply(this, args);
    }
```
please?

Reply:
Done in 7012463. Used `Parameters<Plugins<T>['filter']>` instead of `Parameters<Plugins[]['filter']>` so if the interface args change, so does the implementation.